backend/LexBoxApi/GraphQL/CustomTypes/OrgProjectsGqlConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LexCore.Entities;
+
+namespace LexBoxApi.GraphQL.CustomTypes;
+
+[ObjectType]
+public class OrgProjectsGqlConfiguration : ObjectType<OrgProjects>
+{
+    protected override void Configure(IObjectTypeDescriptor<OrgProjects> descriptor)
+    {
+        descriptor.Field(f => f.Org).Type<NonNullType<OrgGqlConfiguration>>();
```

Comment:
Is it safe to omit this and delete the OrgGqlConfiguration.cs file entirely? Or will we need some special GraphQL setup for orgs?

Reply:
this is here telling hot chocolate that these fields are not nullable, however in C# they are nullable since they might not be included in a sql query, however hot chocolate makes sure it is included if the gql query requests it, so it'll never be null.